dev/view/header.html

```diff
@@ -0,0 +1,9 @@
+<div class="header">
+    <div class="pull-left">
+        Group: &nbsp;&nbsp;
+    </div>
+    <div class="pull-left">
+        <select id="groups-dd"></select>
+    </div>
+    <span class="clearfix" />
+</div>
```

Comment:
Same goes here. Should be a part of `feed.html`.

Reply:
For this one, I thought the header could be reusable if another tab in the future needed that header in it. Will move it back to feed